css/structure/jquery.mobile.collapsible.css

```diff
@@ -97,3 +97,6 @@
 .ui-collapsible-set .ui-collapsible.ui-first-child {
 	margin-top: 0;
 }
+.ui-collapsible-icon-right {
+	float: right;
```

Comment:
To give it the same style as ```ui-widget-icon-floatend``` you also need to add ```bottom: auto;```.
The 2px bottom offset makes it look better when the icon is next to the text but not when the icon is floating.